tests/11_reactions/ref_output/flow_decay_long_gmsh/mass_balance.txt

```diff
@@ -23,483 +23,483 @@
+5.5	"IMPLICIT BOUNDARY"	"E"	2.66322e-16	2.66454e-16	-1.31679e-19	0	0	0	0	0	0	0	0	0
+5.5	"IMPLICIT BOUNDARY"	"F"	2.79449e-16	2.84217e-16	-4.76803e-18	0	0	0	0	0	0	0	0	0
+5.5	"IMPLICIT BOUNDARY"	"G"	2.49241e-16	3.01981e-16	-5.27393e-17	0	0	0	0	0	0	0	0	0
+5.5	"IMPLICIT BOUNDARY"	"H"	3.00928e-16	3.19744e-16	-1.8816e-17	0	0	0	0	0	0	0	0	0
+5.5	"ALL"	"A"	3.49043e-16	3.55271e-16	-6.22876e-18	0.0157792	0	0	0	1.73121e-16	-0.00709298	1.77969e-15	-0.119221	-7.63278e-17
+5.5	"ALL"	"B"	1.99234e-16	2.13163e-16	-1.39289e-17	0.0352858	0	0	0	9.56925e-17	-0.0198834	5.16317e-16	-0.234714	-1.52656e-16
+5.5	"ALL"	"C"	-1.44772e-15	3.73035e-16	-1.82075e-15	4.61247	0	0	0	-7.02328e-16	0.109085	-4.28783e-15	4.20747	-3.55271e-15
+5.5	"ALL"	"D"	2.12063e-16	2.13163e-16	-1.10018e-18	0.00278706	0	0	0	1.05572e-16	-0.00232634	7.75591e-16	-0.537213	-2.68882e-16
+5.5	"ALL"	"E"	2.66322e-16	2.66454e-16	-1.31679e-19	0.000333579	0	0	0	1.33095e-16	-0.000332849	1.19888e-15	-0.674666	-7.04731e-17
+5.5	"ALL"	"F"	2.79449e-16	2.84217e-16	-4.76803e-18	0.0120787	0	0	0	1.38077e-16	-0.00834541	9.08274e-16	-0.797921	-3.81639e-16
+5.5	"ALL"	"G"	2.49241e-16	3.01981e-16	-5.27393e-17	0.133603	0	0	0	1.14851e-16	-0.0494967	3.86746e-16	-0.811397	-7.21645e-16
+5.5	"ALL"	"H"	3.00928e-16	3.19744e-16	-1.8816e-17	0.0476662	0	0	0	1.46199e-16	-0.0216078	9.69428e-16	-1.03233	-1.94289e-16
+6	"region_1"	"A"	0	0	0	0.010641	0	0	0	0	0	0	0	0
+6	"region_1"	"B"	0	0	0	0.022167	0	0	0	0	0	0	0	0
+6	"region_1"	"C"	0	0	0	4.69116	0	0	0	0	0	0	0	0
+6	"region_1"	"D"	0	0	0	0.00150881	0	0	0	0	0	0	0	0
+6	"region_1"	"E"	0	0	0	0.000166972	0	0	0	0	0	0	0	0
+6	"region_1"	"F"	0	0	0	0.00705377	0	0	0	0	0	0	0	0
+6	"region_1"	"G"	0	0	0	0.0952113	0	0	0	0	0	0	0	0
+6	"region_1"	"H"	0	0	0	0.0320872	0	0	0	0	0	0	0	0
+6	"IMPLICIT BOUNDARY"	"A"	3.51071e-16	3.55271e-16	-4.20048e-18	0	0	0	0	0	0	0	0	0
+6	"IMPLICIT BOUNDARY"	"B"	2.04412e-16	2.13163e-16	-8.75034e-18	0	0	0	0	0	0	0	0	0
+6	"IMPLICIT BOUNDARY"	"C"	-1.47878e-15	3.73035e-16	-1.85182e-15	0	0	0	0	0	0	0	0	0
+6	"IMPLICIT BOUNDARY"	"D"	2.12567e-16	2.13163e-16	-5.95597e-19	0	0	0	0	0	0	0	0	0
+6	"IMPLICIT BOUNDARY"	"E"	2.66388e-16	2.66454e-16	-6.59114e-20	0	0	0	0	0	0	0	0	0
+6	"IMPLICIT BOUNDARY"	"F"	2.81433e-16	2.84217e-16	-2.78445e-18	0	0	0	0	0	0	0	0	0
+6	"IMPLICIT BOUNDARY"	"G"	2.64396e-16	3.01981e-16	-3.75843e-17	0	0	0	0	0	0	0	0	0
+6	"IMPLICIT BOUNDARY"	"H"	3.07078e-16	3.19744e-16	-1.26663e-17	0	0	0	0	0	0	0	0	0
+6	"ALL"	"A"	3.51071e-16	3.55271e-16	-4.20048e-18	0.010641	0	0	0	1.74521e-16	-0.0051382	1.95421e-15	-0.124359	-7.80626e-17
+6	"ALL"	"B"	2.04412e-16	2.13163e-16	-8.75034e-18	0.022167	0	0	0	9.9617e-17	-0.0131187	6.15934e-16	-0.247833	-1.38778e-16
+6	"ALL"	"C"	-1.47878e-15	3.73035e-16	-1.85182e-15	4.69116	0	0	0	-7.23859e-16	0.0786976	-5.01169e-15	4.28616	-3.55271e-15
+6	"ALL"	"D"	2.12567e-16	2.13163e-16	-5.95597e-19	0.00150881	0	0	0	1.06031e-16	-0.00127825	8.81622e-16	-0.538491	-3.18972e-16
+6	"ALL"	"E"	2.66388e-16	2.66454e-16	-6.59114e-20	0.000166972	0	
```

Comment:
Ahoj Honzo,

posilam obrazek pro ten test 11, kde se ref. vysledky hodne lisi. Na
obrazku je hmota pro latku B vs. cas. Porovnavano s resenim se zjemnenym
casovym krokem, overil jsem, ze "presne reseni" vyjde stejne jak s puvodni
metodou tak novou.

H.

On 30 April 2016 at 14:42, Jan Brezina <notifications@github.com> wrote:

> In tests/11_reactions/ref_output/flow_decay_long_gmsh/mass_balance.txt
> <https://github.com/flow123d/flow123d/pull/557#discussion_r61669209>:
>
> > +6	"region_1"	"F"	0	0	0	0.00705377	0	0	0	0	0	0	0	0
> > +6	"region_1"	"G"	0	0	0	0.0952113	0	0	0	0	0	0	0	0
> > +6	"region_1"	"H"	0	0	0	0.0320872	0	0	0	0	0	0	0	0
> > +6	"IMPLICIT BOUNDARY"	"A"	3.51071e-16	3.55271e-16	-4.20048e-18	0	0	0	0	0	0	0	0	0
> > +6	"IMPLICIT BOUNDARY"	"B"	2.04412e-16	2.13163e-16	-8.75034e-18	0	0	0	0	0	0	0	0	0
> > +6	"IMPLICIT BOUNDARY"	"C"	-1.47878e-15	3.73035e-16	-1.85182e-15	0	0	0	0	0	0	0	0	0
> > +6	"IMPLICIT BOUNDARY"	"D"	2.12567e-16	2.13163e-16	-5.95597e-19	0	0	0	0	0	0	0	0	0
> > +6	"IMPLICIT BOUNDARY"	"E"	2.66388e-16	2.66454e-16	-6.59114e-20	0	0	0	0	0	0	0	0	0
> > +6	"IMPLICIT BOUNDARY"	"F"	2.81433e-16	2.84217e-16	-2.78445e-18	0	0	0	0	0	0	0	0	0
> > +6	"IMPLICIT BOUNDARY"	"G"	2.64396e-16	3.01981e-16	-3.75843e-17	0	0	0	0	0	0	0	0	0
> > +6	"IMPLICIT BOUNDARY"	"H"	3.07078e-16	3.19744e-16	-1.26663e-17	0	0	0	0	0	0	0	0	0
> > +6	"ALL"	"A"	3.51071e-16	3.55271e-16	-4.20048e-18	0.010641	0	0	0	1.74521e-16	-0.0051382	1.95421e-15	-0.124359	-7.80626e-17
> > +6	"ALL"	"B"	2.04412e-16	2.13163e-16	-8.75034e-18	0.022167	0	0	0	9.9617e-17	-0.0131187	6.15934e-16	-0.247833	-1.38778e-16
> > +6	"ALL"	"C"	-1.47878e-15	3.73035e-16	-1.85182e-15	4.69116	0	0	0	-7.23859e-16	0.0786976	-5.01169e-15	4.28616	-3.55271e-15
> > +6	"ALL"	"D"	2.12567e-16	2.13163e-16	-5.95597e-19	0.00150881	0	0	0	1.06031e-16	-0.00127825	8.81622e-16	-0.538491	-3.18972e-16
> > +6	"ALL"	"E"	2.66388e-16	2.66454e-16	-6.59114e-20	0.000166972	0	
>
> Very big difference, about 60%.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/flow123d/flow123d/pull/557/files/064da987ddcf291c2a75b2f1c2d12cd10c9da488..de8db26500435a5e8ccac97aa9d57ef8088a2059#r61669209>
>